jdeeco-core/src/cz/cuni/mff/d3s/deeco/runtimelog/RuntimeLogRecord.java

```diff
@@ -0,0 +1,69 @@
+package cz.cuni.mff.d3s.deeco.runtimelog;
+
+import java.util.Map;
+import cz.cuni.mff.d3s.deeco.model.runtime.api.ComponentInstance;
+
+/**
+ * This class defines a record that can be logged using the {@link RuntimeLogger}. 
+ * 
+ * @author Dominik Skoda <skoda@d3s.mff.cuni.cz>
+ */
+public class RuntimeLogRecord 
+{
+	/**
+	 * The type of the event being logged. There are no constraints on the types of event.
+	 */
+	private String eventType;
+	/**
+	 * The identifier of the {@link ComponentInstance}
+	 * that logs the record.
+	 */
+	private String componentID;
```

Comment:
This should not be componentID, because some of the event are not related to components, some of the are related more components. Should be only id - and give examples to the comment.